views/symptoms/rashes_meningitus.html

```diff
@@ -0,0 +1,316 @@
+                    <li>has a stiff neck</li>
+                    <li>is bothered by light</li>
+                    <li>seems confused</li>
+                    <li>is shaking uncontrollably</li>
+                    <li>has a fever you can’t control</li>
+                </ul>
+
+            </div>
+
+            <!-- fever and red cheeks -->
+            <div class="image-module">
+
+              <h3>Fever and red cheeks</h3>
+              
+              <img src="/images/rashes/rashes-slapped-cheek.jpg" alt="Image of rash on cheeks" />
+
+              <div class="image-description">
+                  <p>A fever with bright red rash on both cheeks can be <a href="http://www.nhs.uk/conditions/slapped-cheek-syndrome/Pages/Introduction.aspx">slapped cheek syndrome</a>. Your child may have a cold and the rash can spread to the body.</p>
+                  <p>It usually clears up within a week. Children’s paracetamol can bring down a fever.</p>
+              </div>
+
+            </div>
+
+            <!-- Small spots and blisters -->
+            <div class="image-module">
+                
+                <h3>Small spots and blisters</h3>
+
+                <img src="/images/rashes/rashes-chicken-pox.jpg" alt="Image of chickenpox" />
+
+                <div class="image-description">
+                    <p><a href="http://www.nhs.uk/conditions/chickenpox/Pages/Introduction.aspx">Chickenpox</a> causes red spots that turn to blisters.  They can be itchy.  They eventually scab and fall off.  Some children have a few spots, while others have them all over their body.</p>
+                </div>
+
+            </div>
+
+            <!-- Blisters on hands and feet -->
+            <div class="image-module">
+
+                <h3>Blisters on hands and feet and in the mouth. </h3>
+
+                <img src="/images/rashes/rashes-hand-foot-and-mouth.jpg" alt="Image of hand, showing rash from hand foot and mouth" />
+
+                <div class="image-description">
+
+                    <p>A common childhood illness that causes blisters on the hands and feet and ulcers on the tongue, is <a href="http://alpha.nhs.uk/hand-foot-and-mouth-disease">hand, foot and mouth disease</a>. It also causes fever and your child may have a cold.</p>
+                    <p>It usually clears up in about a week. Children’s paracetamol can bring down a fever.</p>
+                </div>
+
+            </div>
+
+            <!-- Pink red rash-->
+            <div class = "image-module">
+                
+                <h3>Pink, red rash</h3>
+
+                <img src="/images/rashes/rashes-scarlet-fever.jpg" alt="Image of scarlet fever rash" />
+
+                <div class="image-description">
+
+                    <p><a href="http://www.nhs.uk/Conditions/Scarlet-fever/Pages/Introduction.aspx">Scarlet fever </a> causes a pink-red rash, which feels like sandpaper and looks like sunburn.</p>
```

Comment:
Remove the whitespace after the anchor so the underline doesn't extend past the word